src/ansys/mechanical/core/embedding/app.py

```diff
@@ -34,6 +34,7 @@
 from ansys.mechanical.core.embedding.addins import AddinConfiguration
 from ansys.mechanical.core.embedding.appdata import UniqueUserProfile
 from ansys.mechanical.core.embedding.imports import global_entry_points, global_variables
+from ansys.mechanical.core.embedding.messages import MessageManager
```

Comment:
can you import this lazily? I think pandas does something on initializatino which might slow things down